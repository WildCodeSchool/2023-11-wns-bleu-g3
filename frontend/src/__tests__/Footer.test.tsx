import { render } from "@testing-library/react";
import Footer from "@/components/footer"

describe("Footer component", () => {
  it("renders correctly", () => {
    const view = render(
      <Footer/>
    );

    expect(view.baseElement).toMatchSnapshot();
  });
});