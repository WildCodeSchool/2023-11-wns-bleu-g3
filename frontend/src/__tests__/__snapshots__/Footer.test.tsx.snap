// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<body>
  <div>
    <footer
      class="bg-anchor bg-opacity-80 flex flex-col md:flex-row justify-center md:justify-between items-center p-4 gap-6 md:gap-0"
    >
      <div
        class="flex flex-col md:flex-row justify-center items-center gap-2"
      >
        <img
          alt=""
          class="w-14"
          src="../../img/greenfoot-logo.png"
        />
        <div
          class="flex flex-col justify-center md:items-start items-center"
        >
          <h3
            class="font-bold text-shore drop-shadow-lg text-xl md:text-3xl"
          >
            GreenFoot
          </h3>
          <h4
            class="text-white font-normal"
          >
            Vers un futur viable.
          </h4>
        </div>
      </div>
      <ul
        class="flex flex-col justify-center items-center text-white"
      >
        <li
          class="cursor-pointer hover:underline"
        >
          <a
            href="#"
          >
            Contactez-nous
          </a>
        </li>
        <li
          class="cursor-pointer hover:underline"
        >
          <a
            href="#"
          >
            Mentions l√©gales
          </a>
        </li>
      </ul>
    </footer>
  </div>
</body>
`;
